process-automation/latest/model/data-models.md

```diff
@@ -0,0 +1,194 @@
+
+5. Enter a meaningful title.
+
+    The title entered will be seen by a user when using a form with this data model.
+
+6. Click the **+** icon to add a property to the Person object.
+
+    A new row is created.
+
+6. Enter `Name` for the new row and then select the **Required** check box.
+
+    Selecting the required check box ensures when a new person is added, the entry must include a Name.
+
+7. From the **Type** dropdown list select **string** and then enter a meaningful title.
+
+8. Click the **+** icon to add another property to the Person object.
+
+9. Enter `DateOfBirth` for the new row.
+
+10. From the **Type** dropdown list select **date** from the Modelling app section.
+
+11. Click the **+** icon to add another property to the Person object.
+
+12. Enter `Gender` for the new row and then select the **Required** check box.
+
+13. From the **Type** dropdown list select **enum** from the Enumeration section and then enter a meaningful title.
+
+14. Select the cog icon to access the Advanced settings and enter the gender definitions you would like to include in the **Enumerated values** box. Click **Save**.
+
+    You can see the JSON definition of the enumerated values in the preview section.
+
+15. Click the **Save** icon to save the data model and then to see the JSON file created of the data model, click the **JSON Editor** button .
+
+  **Note:** You can also use references in your data model, for more see [Using Ref in a data model](#using-ref-in-a-data-model).
+
+```JSON
+{
+    "description": "This model provides the data structure for the object Person.",
+    "type": "object",
+    "properties": {
+        "Name": {
+            "type": "string",
+            "title": "Name of person"
+        },
+        "DateOfBirth": {
+            "$ref": "#/$defs/primitive/date"
+        },
+        "Gender": {
+            "enum": [
+                "male",
+                "female"
+            ],
+            "title": "Gender"
+        }
+    },
+    "title": "Person",
+    "required": [
+        "Name"
+    ]
+}
+```
+
+You have created a data model called Person that can be used in Process Automation. Use the Simulator on the right to view how the information you have added to the data model appears on a form. You can test the rules you have created for each property and you can see the JSON output when a new person is added.
+
+![Data model]({% link process-automation/images/data-model.png %})
+
+### Using a data model in a project
```

Comment:
```suggestion
### Using a data model in a process
```
Is a good example on how to use it in a process but as mentioned before, the data model can be used in every variable definition (scripts, forms,...)

Reply:
changed